tailwind.config.js

```diff
@@ -3,7 +3,9 @@
 export default {
   content: ["./*.html", "./src/**/*.html"],
   theme: {
-    extend: {},
+    extend: {
+      
+    },
```

Comment:
Useless change...
Prevent any useless changes & config your IDE according to project settings.

```suggestion
    extend: {},
```